corehq/blobs/management/commands/check_blob_logs.py

```diff
@@ -8,12 +8,14 @@
 from couchdbkit.exceptions import ResourceNotFound
 from couchexport.models import SavedBasicExport
 from django.core.management import BaseCommand, CommandError
+from gevent.pool import Pool
 
 import corehq.apps.accounting.models as acct
 import corehq.apps.app_manager.models as apps
 import corehq.apps.hqmedia.models as hqmedia
 from corehq.apps.export import models as exports
 from corehq.blobs import get_blob_db, CODES
+from corehq.blobs.management.commands.run_blob_migration import set_max_connections
```

Comment:
Would it be worth moving `set_max_connections` as a util and then imported to the two management commands using it?
My habit is to avoid sharing code between management commands, i like to see them as standalone commands not sharing within themselves.

Reply:
https://github.com/dimagi/commcare-hq/pull/22995